package.json

```diff
@@ -5,7 +5,11 @@
   "description": "rich text editor built with React and ProseMirror",
   "main": "dist/index.js",
   "scripts": {
-    "build_bin": "node utils/build_bin.js",
+    "dist:css": "cp src/ui/*.css dist/ui && cp src/ui/mathquill-editor/*.css dist/ui/mathquill-editor",
```

Comment:
Naming was a concern here. Thoughts?

Reply:
maybe `bin:demos`? 
I'd like to keep the script to build static demo HTMLs until we have real site for the DEMOs.

Reply:
I'll update in the next PR with "build:demos"